COMETwebapp/Components/ParameterTable.razor

```diff
@@ -47,26 +48,44 @@ Copyright (c) 2022 RHEA System S.A.
     /// </summary>
     readonly ObservableCollection<BuiltParameter> Parameters = new ObservableCollection<BuiltParameter>();
 
+    /// <summary>
+    /// Listeners for the components to update it with ISession
+    /// </summary>
+    private Dictionary<string, IDisposable> Listeners = new Dictionary<string, IDisposable>();
+
     /// <summary>
     /// Initialize component at first render and after session update
     /// </summary>
     protected override void OnInitialized()
     {
         this.InitializeParameters();
 
-        CDPMessageBus.Current.Listen<SessionEvent>().Where(x => x.Status == SessionStatus.EndUpdate).Subscribe(x => {
-            this.Parameters.Clear();
-            this.InitializeParameters();
-        });
+        if (!Listeners.Keys.Contains("SessionEvent"))
+        {
+            Listeners.Add("SessionEvent", CDPMessageBus.Current.Listen<SessionEvent>().Where(x => x.Status == SessionStatus.EndUpdate).Subscribe(x =>
+            {
+                this.Parameters.Clear();
+                this.InitializeParameters();
+            }));
+        }
+    }
+
+    /// <summary>
+    /// Stop and clear listeners of the component
+    /// </summary>
+    public void Dispose()
+    {
+        Listeners.Values.ToList().ForEach(l => l.Dispose());
+        Listeners.Clear();
     }
 
     /// <summary>
     /// Initialize the <see cref="Parameters"> list
     /// </summary>
     private void InitializeParameters()
     {
-        var ListElements = ISessionAnchor.OpenIteration.Element;
-        ListElements.ForEach(element =>
+        var ListElements = ISessionAnchor.GetIteration()?.Element;
```

Comment:
please rename the variable to `elementDefinitions`, no need to put the word `List` in the name of the variable